model-archiver/model_archiver/manifest_components/model.py

```diff
@@ -27,6 +27,12 @@ def __init__(self, model_name, serialized_file, handler, model_file=None, model_
         else:
             self.handler = handler.split("/")[-1]
         self.requirements_file = requirements_file
+        self.config_file = None
+        if config_file:
+            if sys.platform.startswith('win32') and config_file.find("\\") != -1:
```

Comment:
you should be able to use `os.path.basename(path)` to unify logic across windows and linux